README.md

```diff
@@ -2,7 +2,7 @@
 
 Gretel Blueprints provide example Gretel Configurations, Notebooks, and other utilities for solving specific use cases and data challenges.
 
-Blueprint contents and any supporting modules within this repository are licensed under Apache 2.0.  Many Blueprints also will install our [gretel-client](https://github.com/gretelai/gretel-python-client) and [gretel-synthetics](https://github.com/gretelai/gretel-synthetics) open source packages.
+Blueprint contents and any supporting modules within this repository are licensed under Apache 2.0. Many Blueprints also will install our [gretel-client](https://github.com/gretelai/gretel-python-client) and [gretel-synthetics](https://github.com/gretelai/gretel-synthetics) open source packages.
```

Comment:
> open source packages

Should this be SAL or Software Available Licensed packages?

Reply:
I talked to John and he said this one is fine as is.